afdko/PKGBUILD

```diff
@@ -3,32 +3,34 @@
 
 pkgname=afdko
 pkgver=4.0.1
-pkgrel=1
+pkgrel=2
 pkgdesc='Adobe Font Development Kit for OpenType'
 arch=(x86_64)
 url="https://github.com/adobe-type-tools/$pkgname"
-license=(custom)
+license=(Apache-2.0)
 _py_deps=(booleanoperations
           brotli # for fonttools[woff]
           defcon
           fontmath
-          fontparts
           fontpens # for defcon[pens]
           fonttools
           fs # for fonttools[ufo]
-          lxml # for fonttools[lxml] and defcon[lxml]
+          lxml # for defcon[lxml] and fonttools[lxml]
           tqdm
           ufonormalizer
           ufoprocessor
           unicodedata2 # for fonttools[unicode]
           zopfli) # for fonttools[woff]
-depends=(python
-        "${_py_deps[@]/#/python-}"
-        psautohint)
+depends=(gcc-libs
+         glibc
+         libxml2
```

Comment:
How did you confirm this is an actual direct dependency? As a policy Arch packages don't rely on transitive dependencies and do explicitly list dependencies, but that only applies if they are *used* directly. In this case I thought this was only being *used* transitively through python-lxml and hence the dependency only needed to be transitive as well. Did that change?

Reply:
The c/c++ code embedded in the AFDKO package does link against libxml2. That's been true for quite a while -- I believe more than a year now.

Reply:
The error was identified by namcap.